content/post/000-ipfs-in-web-browsers.md

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
```suggestion
---
date: 2019-10-07
url: 2019-10-07-ipfs-browsers-update
title: IPFS Browser Update
author: Dietrich Ayala
---
```
Remember to have this at the top of the file! Put in some placeholder content but feel free to edit